Webster/PandasCSVReport.ipynb

```diff
@@ -79,25 +79,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 139,
+   "execution_count": 7,
    "id": "ccc095d7-7574-4a42-9adf-47a976b64b87",
    "metadata": {},
    "outputs": [],
    "source": [
     "# Filter data by space (sq ft)\n",
     "def filter_by_space(data, min_size, max_size):\n",
-    "        # Remove commas and convert the 'Exhibit Meeting Area' column to integers\n",
-    "    data['Exhibit Meeting Area'] = data['Exhibit Meeting Area'].str.replace(',', '').astype(int)\n",
-    "    \n",
     "    # Apply the filter without int()\n",
-    "    filtered_data = data[(data['Exhibit Meeting Area'] >= min_size) & (data['Exhibit Meeting Area'] <= max_size)]\n",
+    "    filtered_data = data[(data['Exhibit Meeting Area'].str.replace(',', '').astype(int) >= min_size) & (data['Exhibit Meeting Area'].str.replace(',', '').astype(int) <= max_size)]\n",
```

Comment:
Before you were trying to compare string to int. I added .str.replace(',', '').astype(int) to remove commas and to int so the comparison works

Reply:
This fixed the filtering.